src/general/MainMenu.cs

```diff
@@ -468,6 +499,11 @@ private void RunMenuSetup()
         UpdateStoreVersionStatus();
         UpdateLauncherState();
 
+        if (!Settings.Instance.IsNoticePermanentlyDismissed(DismissibleNotice.LowPerformanceWarning))
```

Comment:
I think this if is unnecessary as the actual condition check later will re-check this each frame. Also could the variable be just initialized to true or is there a reason why only after RunMenuSetup it would need to be set to true?